tests/unit/geom/CoordinateArraySequenceFactoryTest.cpp

```diff
@@ -79,7 +79,7 @@ namespace tut
 			ensure( 0 != col);
 
 			const size_t size0 = 0;
-			CoordinateSequencePtr sequence = factory->create(col);
```

Comment:
Was this the API breakage, the need to pass -1, -1 ? Should they be default values instead, to retain API compatibility ? Do they need to be integer rather than booleans ?